custom_components/matter_experimental/entity.py

```diff
@@ -23,6 +23,7 @@ class MatterEntity(entity.Entity):
     def __init__(self, device: MatterDevice, mapping: DeviceMapping) -> None:
         self._device = device
         self._device_mapping = mapping
+        self._entity_description = mapping.entity_description
```

Comment:
The attribute seems to be without underline (`self.entity_description`). With that Sensor entities look much better now. It seems that some mappings (e.g. Light) don't have that attribute set, leading to `NoneType` access. So it should be set conditionally:

```suggestion
        if mapping.entity_description:
            self.entity_description = mapping.entity_description
```